### DISCUSSION TITLE
Tags

### DISCUSSION THREAD

**Discussion by MaksGill** [outline/outline#4671] [General] [answered]
Please tell me guys, is it possible to add tags to OutlineWiki to search for pages? If there is, how to do it?

**Comment by tommoor** [comment, marked as answer]
It is not possible at this time

**Reply by MaksGill** [reply]
OK, thanks

**Reply by danfoxley** [reply]
Oh, that is disappointing. I see plenty of nods and upvotes in Discussion for adding tags (filtering and Kanban/Tables).
